docs/src/api/class-response.md

```diff
@@ -22,6 +22,11 @@ Waits for this response to finish, returns always `null`.
 
 Returns the [Frame] that initiated this response.
 
+## method: Response.fulfilledByServiceWorker
```

Comment:
I'd call it 'fromServiceWorker' to be consistent with the devtools.

Reply:
good call! Also easier to spell!